Mathlib/Analysis/NormedSpace/Exponential.lean

```diff
@@ -447,153 +472,161 @@ theorem expSeries_summable (x : 𝔸) : Summable fun n => expSeries 𝕂 𝔸 n
   (norm_expSeries_summable x).of_norm
 #align exp_series_summable NormedSpace.expSeries_summable
 
-theorem expSeries_summable' (x : 𝔸) : Summable fun n => (n !⁻¹ : 𝕂) • x ^ n :=
-  (norm_expSeries_summable' x).of_norm
+variable (𝕂) in
```

Comment:
Where does `𝕂` appear in this theorem?  I only see it in the body, in which case it could naively be `ℚ`.

Reply:
Ah, right, characteristic p.

Reply:
Actually, no, this theorem only holds in characteristic 0.

Reply:
This lemma transitively uses `norm_nsmul`, which I can't apply here.